projects/imgix-angular-example/src/test.ts

```diff
@@ -12,7 +12,9 @@ declare const require: any;
 // First, initialize the Angular testing environment.
 getTestBed().initTestEnvironment(
   BrowserDynamicTestingModule,
-  platformBrowserDynamicTesting(),
+  platformBrowserDynamicTesting(), {
+    teardown: { destroyAfterEach: false }
```

Comment:
Same as above, I'm Angular ignorant, but for posterity, what does setting this to `false` solve?

Reply:
Angular 13 sets the `teardown` flag to `true` by default for `TestBed`; for compatibility reasons, during the Angular 12 to 13 update process `ng update` opts out of the new teardown behavior.